packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -533,9 +577,17 @@ export class RemoteMediaManager extends EventsScope {
     for (let i = 0; i < this.config.audio.numOfActiveSpeakerStreams; i += 1) {
       // eslint-disable-next-line no-await-in-loop
       const slot = await this.receiveSlotManager.allocateSlot(MediaType.AudioMain);
-
       this.slots.audio.push(slot);
     }
+    // create slot for interpretation language audio
+    if (this.config.receiveNamedMediaGroup.value) {
+      const siSlot = await this.receiveSlotManager.allocateSlot(
+        MediaType.AudioMain,
+        this.config.receiveNamedMediaGroup
+      );
+
+      this.slots.audio.push(siSlot);
```

Comment:
I'm wondering if maybe we should have a separate `this.slots.siAudio` instead of appending it to the end of `this.slots.audio` array. It doesn't make much difference for now, but feels like it would be cleaner.

Reply:
Do you mean that you want to add a new parameter in the constructor of the RemoteMeidaGroup class?

Reply:
no, just having a separate siAudio here in remoteMediaManager, so `this.slots.siAudio = siSlot;`